packages/run-protocol/test/swingsetTests/governance/vat-voter.js

```diff
@@ -53,7 +64,14 @@ const build = async (log, zoe) => {
             } governor instance`,
           );
 
-          assertBallotConcernsParam(issue.paramSpec, ballotDetails);
+          assertBallotConcernsParam(
+            {
+              // FIXME should be `key: issue.key` but that fails
```

Comment:
wasn't sure what to do here @Chris-Hibbert

Reply:
For the AMM, `paramSpec` is `{ collateralBrand: brand }`.   It looks like `assertBallotConcernsParam()` might be failing here at master, but without being noticed. I'm not sure why it wouldn't be detected.